modules/microshift-lvms-system-requirements.adoc

```diff
@@ -11,15 +11,17 @@ Using LVMS in {microshift-short} requires the following system specifications.
 [id="lvms-volume-group-name_{context}"]
 == Volume group name
 
-The default integration of LVMS selects the default volume group (VG) dynamically. If there are no volume groups on the {microshift-short} host, LVMS is disabled.
+If the LVMS configuration is not present at `/etc/microshift/lvmd.yaml`, MicroShift will attempt to assign a default volume group dynamically using the `vgs` command.
+
+If there are no volume groups on the {microshift-short} host, LVMS is disabled.
 
 If there is only one VG on the {microshift-short} host, that VG is used. If there are multiple volume groups, the group `microshift` is used. If the `microshift` group is not found, LVMS is disabled.
 
 If you want to use a specific VG, LVMS must be configured to select that VG. You can change the default name of the VG in the configuration file. For details, read the "Configuring the LVMS" section of this document.
 
 You can change the default name of the VG in the configuration file. For details, read the "Configuring the LVMS" section of this document.
 
-Prior to launching, the `lvmd.yaml` configuration file must specify an existing VG on the node with sufficient capacity for workload storage. If the VG does not exist, the node controller fails to start and enters a `CrashLoopBackoff` state.
+After MicroShift has started, the lvmd.yaml may be updated to include or remove VGs.  For the changes to take effect, restart the MicroShift service. If the `lvmd.yaml` is deleted, LVMS will then be disabled.
```

Comment:
```suggestion
After the {microshift-short} starts, the `lvmd.yaml` can be updated to include or remove VGs. To implement changes, you must restart {microshift-short}. If the `lvmd.yaml` is deleted, LVMS is disabled.
```
"If the `lvmd.yaml` is deleted, LVMS is disabled." Is that true, or does MicroShift try to auto-assign a VG when it restarts (as we said in the beginning)?

If we don't have a VG with enough capacity for workload storage, do we still get the error? Remember that this section of the docs is "Using LVMS in MicroShift requires the following system specifications." We're telling users how to use LVMS, we are *not* describing the default state of the system. If an error occurs when the storage isn't configured correctly, we should still list that somewhere.